### PR TITLE
chore(rebranding): RRPproxy to CentralNic Reseller

### PR DESCRIPTION
**Pull request by AsifNawaz-cnic**

### CODE REVIEW

File: modules/addons/cnicssl_addon/cnicssl_addon.php [hunk 1]
```diff
@@ -67,17 +67,17 @@ function cnicssl_addon_output(array $vars): void
 {
     global $templates_compiledir;
 
-    if ($vars["registrar"] === "RRPproxy") {
-        if (!class_exists('\WHMCS\Module\Registrar\RRPproxy\APIClient')) {
+    if ($vars["registrar"] === "CNIC") {
+        if (!class_exists('\WHMCS\Module\Registrar\keysystems\APIClient')) {
```
Comment (KaiSchwarz-cnic) [inline modules/addons/cnicssl_addon/cnicssl_addon.php:71]: Wouldn't it be `\WHMCS\Module\Registrar\Keysystems\APIClient`?

**Review by KaiSchwarz-cnic — CHANGES_REQUESTED** [review]

File: modules/addons/cnicssl_addon/cnicssl_addon.php (hunk 1 above)
Comment (AsifNawaz-cnic) [reply modules/addons/cnicssl_addon/cnicssl_addon.php]: I have fixed the requested changes, thanks for notifying

**Comment by KaiSchwarz-cnic** [comment]
@AsifNawaz-cnic we need the workflow changes in place without the rebranding. Otherwise we cannot merge the workflow changes before august 31st.

**Review by KaiSchwarz-cnic — CHANGES_REQUESTED** [review]
See my last comment 🙋

**Comment by AsifNawaz-cnic** [comment]
> @AsifNawaz-cnic we need the workflow changes in place without the rebranding. Otherwise we cannot merge the workflow changes before august 31st.

I will create a separate pull request for workflow, thanks for reminder.

**Pull request closed without merging**